controllers/providers/aws/aws.go

```diff
@@ -238,3 +241,12 @@ func GetInstanceTypeNetworkInfo(instanceTypes []*ec2.InstanceTypeInfo, instanceT
 	}
 	return nil
 }
+
+func GetInstanceTypeArchetectures(instanceTypes []*ec2.InstanceTypeInfo, instanceType string) []*string {
+	for _, instanceTypeInfo := range instanceTypes {
+		if aws.StringValue(instanceTypeInfo.InstanceType) == instanceType {
+			return instanceTypeInfo.ProcessorInfo.SupportedArchitectures
```

Comment:
Would be good if this can return a string value slice. you can use `return aws.StringValueSlice(instanceTypeInfo.ProcessorInfo.SupportedArchitectures)`